src/OrchardCore.Themes/TheAdmin/Assets/scss/TheAdmin.scss

```diff
@@ -1392,4 +1392,11 @@ input[type=password]::-ms-reveal,
 input[type=password]::-ms-clear
 {
     display: none;
+}
+
+.sticky {
+    position: -webkit-sticky;
```

Comment:
maybe rename this to .action-bar and use position-sticky from BS4. Remove the position: -webkit-sticky and position sticky.

Reply:
Make sense